test/fixtures/cookbooks/test/recipes/package.rb

```diff
@@ -1,7 +1,17 @@
 # frozen_string_literal: true
 apt_update
 
-haproxy_install 'package'
+case node['platform_family']
+when 'amazon', 'rhel'
+  enable_uis = true
+else
+  enable_uis = false
+end
+
+haproxy_install 'package' do
+  package_name enable_uis ? 'haproxy18u' : 'haproxy'
```

Comment:
The centos-7 test seems to think the haproxy package isn't installed...it totally is:
```
rpm -qa | grep haproxy
haproxy18u-1.8.19-1.ius.centos7.x86_64
```

Reply:
gah it's because the package name is actually called `haproxy18u`

Reply:
https://github.com/iuscommunity-pkg/haproxy18u